### PR TITLE
Bump AH and BH runtime versions

### PR DESCRIPTION
**Pull request by al3mart**
After merging https://github.com/paritytech/polkadot-sdk/pull/2522 asset-hub-rococo and bridge-hub-rococo runtime versions need to be bumped for their deployment.

### CODE REVIEW

**Review by alistair-singh — APPROVED** [review]
+1

**Review by acatangiu — APPROVED** [review]

File: cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs [hunk 1]
```diff
@@ -111,7 +111,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("statemine"),
 	impl_name: create_runtime_str!("statemine"),
 	authoring_version: 1,
-	spec_version: 1_005_000,
+	spec_version: 1_005_100,
```
Comment (joepetrowski) [inline cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs:114]:
```suggestion
	spec_version: 1_005_001,
```

File: cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs [hunk 2]
```diff
@@ -209,7 +209,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("bridge-hub-rococo"),
 	impl_name: create_runtime_str!("bridge-hub-rococo"),
 	authoring_version: 1,
-	spec_version: 1_005_000,
+	spec_version: 1_005_100,
```
Comment (joepetrowski) [inline cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs:212]:
```suggestion
	spec_version: 1_005_001,
```

**Review by joepetrowski — COMMENTED** [review]
Was transaction version checked?

**Review by Lederstrumpf — APPROVED** [review]

**Review by bkontur — APPROVED** [review]

**Comment by al3mart** [comment]
> Was transaction version checked?

@claravanstaden @alistair-singh Could you help me figure out if any function signature from existing extrinsics or any indices from existing pallets or calls, have been modified ?

We might want some CI automation for this whenever a pallet or a runtime is modified - I know we were running these checks prior to a release so far

**Comment by alistair-singh** [comment]
> > Was transaction version checked?
> 
> @claravanstaden @alistair-singh Could you help me figure out if any function signature from existing extrinsics or any indices from existing pallets or calls, have been modified ?
> 
> We might want some CI automation for this whenever a pallet or a runtime is modified - I know we were running these checks prior to a release so far

Confirming now.

**Comment by alistair-singh** [comment]
> > > Was transaction version checked?
> > 
> > 
> > @claravanstaden @alistair-singh Could you help me figure out if any function signature from existing extrinsics or any indices from existing pallets or calls, have been modified ?
> > We might want some CI automation for this whenever a pallet or a runtime is modified - I know we were running these checks prior to a release so far
> 
> Confirming now.

Went through the PR again and no modifications to existing extrinsic signatures and no pallet/calls index changes. Only new pallets added.

**Pull request merged**